### ISSUE TITLE
EventManager Initialization

### ISSUE THREAD

**Issue by benbroadley88** [zendframework/zend-mvc#206]
Hello - brand new to Zend so early apologies if I'm just being silly...

I'm trying to follow the Zend-MVC [intro documentation](https://docs.zendframework.com/zend-mvc/intro/#bootstrapping-an-application) which suggests that to bootstrap an application, the following should be run: 
```
$serviceManager->setFactory('EventManager', function ($serviceManager) {
    $eventManager = new EventManager();
    $eventManager->setSharedManager($serviceManager->get('SharedEventManager'));
    return $eventManager;
});
```

However, when I tried this I'm getting a Fatal Error as the setSharedManager method doesn't appear to exist. I took a look at the source, and as far as I can tell, the only way to set the SharedEventManger is in the constructor. Would I be correct in thinking that the documentation should instead read as follows?

```
$serviceManager->setFactory('EventManager', function ($serviceManager) {
    return new EventManager($serviceManager->get('SharedEventManager'));
});
```

**Comment by weierophinney** [comment]
Indeed; we missed that when updating the docs for the new version. Would you be willing to create a pull request with that change?

**Comment by thomasvargiu** [comment]
The documentation is not updated for zend-mvc and zend-eventmanager 3.0.

Look at the [zend-mvc](https://docs.zendframework.com/zend-mvc/migration/to-v3-0/#application-class) and [zend-eventmanager](https://zendframework.github.io/zend-eventmanager/migration/changed/#eventmanager__construct) migration guides for changes.

**Comment by benbroadley88** [comment]
@weierophinney Yeah no problem - I will submit that shortly.

**Comment by Xerkus** [comment]
Fixed in #207
